### PR TITLE
Update sbc-common-component version number

### PR DESCRIPTION
**Pull request by eve-git**
Update sbc-common-component version number

*Issue #:* /bcgov/entity#6682

*Description of changes:*
Update sbc-common-component version number

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by eve-git** [comment]
Unit test is passed. Ready to merge.

**Comment by severinbeauvais** [comment]
^^ I'm not worrying about the code coverage check.

**Review by severinbeauvais — APPROVED** [review]
LGTM!

No issues with testing? Do you want other reviewers, or can I merge this?

**Comment by severinbeauvais** [comment]
^^ I don't have a problem with the code coverage check.

**Review by severinbeauvais — CHANGES_REQUESTED** [review]
See my new comment.

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "provinces": "^1.11.0",
     "regenerator-runtime": "^0.13.3",
     "register-service-worker": "^1.6.2",
-    "sbc-common-components": "2.4.16",
+    "sbc-common-components": "^2.4.18",
```
Comment (severinbeauvais) [inline package.json:36]: Remove ^.

Comment (eve-git) [reply package.json]: done

**Review by severinbeauvais — APPROVED** [review]
LGTM.

@andrepestana-aot If you approve, please merge.

**Review by andrepestana-aot — APPROVED** [review]
👍

**Pull request merged**